index.bs

```diff
@@ -2301,7 +2301,16 @@ SHOULD be aborted.
     See [WHATWG HTML WG Issue #2711](https://github.com/whatwg/html/issues/2711) for more details. 
 
 
-## Authentication Extensions Client Inputs (typedef {{AuthenticationExtensionsClientInputs}}) ## {#iface-authentication-extensions-client-inputs}
+## WebAuthn Extensions Inputs and Outputs
+
+The subsections below define the data types used for conveying [=WebAuthn extension=] inputs and outputs.
+
+Note: [=Authenticator extension outputs=] are conveyed as a part of [=Authenticator data=] (see [Table 1](#table-authData)).
+
+Note: The types defined below&mdash;{{AuthenticationExtensionsClientInputs}}, {{AuthenticationExtensionsClientOutputs}}, and {{AuthenticationExtensionsAuthenticatorInputs}}&mdash;are applicable to both [=registration extensions=] and [=authentication extensions=]. The "Authentication..." portion of their names should be regarded as meaning "WebAuthentication..."
```

Comment:
I would want to s/"$/"./ but I think that goes against "correct" writing style...

Reply:
LOL, yeah, I thought about that too :)   I don't think it matters either way ;-)